### ISSUE TITLE
adapt foreground segmentation for ACDC segmentation

### ISSUE THREAD

**Issue by LevanBokeria** [alan-turing-institute/heartfm-evals#6]
Status quo:

Currently downloaded and preprocessed ACDC using CineMA scripts. 
Considering whether all the transforms should still be applied when loading ACDC to train the linear classifier.
Considering which dataloaders to re-use from CineMA to load the ACDC images. 
Considering whether to subdivide the training set into train/val set, or use k-fold validation.
Afterwards will:
- Load the ACDC images.
- Visualize the raw and labeled masks.
- Transform to be ready for DINO: eg, add 3 channels, resize and quantize for 16x16 patches.
- Extract DINO features.
- Adapt the classifier training to use grid search for hyperparameters. 
- Train the classifier.
- Evaluate on test set.
- Visualize.
